src/app.ts

```diff
@@ -1,25 +1,28 @@
 import express, { Request, Response, NextFunction } from "express";
 import nunjucks from "nunjucks";
 import path from "path";
-import logger from "./lib/Logger";
+import { logger } from "./utils/logger";
 import routerDispatch from "./router.dispatch";
+import { env } from './config';
 
 const app = express();
 
 // const viewPath = path.join(__dirname, "/views");
 app.set("views", [
     path.join(__dirname, "/views"),
-    path.join(__dirname, "/../node_modules/govuk-frontend")
+    path.join(__dirname, "/../node_modules/govuk-frontend"),
+    path.join(__dirname, "../node_modules/govuk-frontend"), // This if for when using ts-node since the working directory is src
+    path.join(__dirname, "node_modules/govuk-frontend/components")
 ]);
```

Comment:
These template paths don't work and result in an error when accessing the service.
```suggestion
app.set("views", [
    path.join(__dirname, "views"),
    path.join(__dirname, "node_modules/govuk-frontend"),
    path.join(__dirname, "../node_modules/govuk-frontend"), // This if for when using ts-node since the working directory is src
    path.join(__dirname, "node_modules/govuk-frontend/components")
]);
```